tests/Unit/TestHelpers.hpp

```diff
@@ -226,6 +218,17 @@ T serialize_and_deserialize(const T& t) {
   return deserialize<T>(serialize<T>(t).data());
 }
 
+/// \ingroup TestingFrameworkGroup
+/// \brief Tests the serialization of comparable types
+/// \example
+/// To test serialization of comparable types use:
+/// \snippet Test_PupStlCpp11.cpp example_serialize_copyable
```

Comment:
Does that snippet still exist?

Reply:
sorry it should be renamed example_serialize_comparable